css-alpha-channel/index.html

```diff
@@ -0,0 +1,79 @@
+---
+feature_name: Alpha Channel Support in CSS Color Parsing
```

Comment:
This should be something like "Alpha Channel Values using Hex Colors" or something along those lines, to indicate that it's just the hex part that's new.